src/management-system-v2/app/(dashboard)/[environmentId]/processes/[processId]/modeler-share-modal-option-public-link.tsx

```diff
@@ -25,7 +23,7 @@ const ModelerShareModalOptionPublicLink = ({
   const environment = useEnvironment();
 
   const [token, setToken] = useState<String | null>(null);
-  const [isShareLinkChecked, setIsShareLinkChecked] = useState(shared);
+  const [isShareLinkChecked, setIsShareLinkChecked] = useState(shareTimestamp > 0);
```

Comment:
Same as above. I think this does not need to be a state. If you want to avoid constant checks for `shareTimestamp > 0` you could just do `const isShareLinkChecked = shareTimestamp > 0`. This way you can remove all the instances where you have to set this state variable.